### ISSUE TITLE
ImportError: cannot import name titanic

### ISSUE THREAD

**Issue by mncvnn** [tflearn/tflearn#249]
Hi guys,
I'm facing with this error right now. Not sure how to fix it, because this is the first time I check out TFLearn - Quick Start.
>  [ ~/Desktop ] 👉  python my_test.py
> Traceback (most recent call last):
>   File "my_test.py", line 7, in <module>
>     from tflearn.datasets import titanic
> ImportError: cannot import name titanic

I'm running a mac (10.11.6) and follow the installation tutor already.
Thanks

**Comment by braddengross** [comment]
Which version of TFLearn are you using? It looks like the titanic dataset code was only added a week ago and may not be in your version unless you pulled it directly from github.

**Comment by mncvnn** [comment]
Oh,
I'm using stable version, not from the git version.
Maybe I'll try upgrade tflearn and let you known later.
Thanks.

**Comment by braddengross** [comment]
Sure, you can check your source and just look for the __init__.py file under tflearn/datasets and that will show you which datasets your version has.

**Comment by citizenrich** [comment]
Same error with git version.

**Comment by boggogo** [comment]
Same error here as well. Latest version of Tensorflow and the stable version of the TFLearn library

**Comment by aymericdamien** [comment]
Could you try:
```
pip install --upgrade git+https://github.com/tflearn/tflearn.git
```
it is working well for me, I am not sure what is wrong here. I will update stable version soon, so that pip install tflearn will be enough.

**Comment by citizenrich** [comment]
odd. I had done that a few hours ago but now it's working for me. happy it's working! loving the tflearn magic.

**Comment by ovcharenkoo** [comment]
Solution proposed by @aymericdamien worked out this problem, thanks!

**Comment by austintrombley** [comment]
This fixes the issue for the titanic, but it breaks the load_csv:  ensuing error: ImportError: cannot import name load_csv

**Comment by KocholoDarShahr** [comment]
I have the same issue. I tried both stable and bleeding edge versions and yet cannot import titanics! Any suggestion?

**Comment by JiawenQi98** [comment]
Same error here~ ImportError: cannot import name 'load_csv'

**Comment by bobbyangelov** [comment]
@JiawenQi98 try the solution from @aymericdamien . Had the problem today, and that solved it.

**Comment by Relunya** [comment]
I also have the import problem for load-csv now: ImportError: cannot import name 'load_csv'
One month ego it was working!
I tried solution by @aymericdamien but it didn't work for me.
I also tried pip install tflearn and it did not work either!
restart kernel many times, no difference!

**Comment by pmunin** [comment]
Error still happens on default installation (following installation instructions in documentation).
`pip install --upgrade git+https://github.com/tflearn/tflearn.git` command is working workaround

**Comment by RoyGithubAccount** [comment]
Is the problem linked to the version of Python we used? I set up TFLearn and TensorFlow with a virtuql environment and used pip3 for python3. However, I notice the TFLearn titanic data is under python 2.7 in my Tensorflow directory. What is the best thing to do so that the call finds the TFLearn titanic dataset? Move the directory in to Python 3.5 or change or add TFLearn to the sys path?

**Comment by RoyGithubAccount** [comment]
Fixed it: 
1.    Use pip3 rather than pip to install TFLearn
$ sudo -H pip3 install --upgrade git+https://github.com/tflearn/tflearn.git 
(I had to use sudo and -H as I was refused permission -- that might be more about my newbie status than anything else though (;-)
2.   Copy Python3.5 directory in to tensorflow/lib next to the Python2.7
I am not sure why the download of tensorflow did not put python3.5 in the lib directory. It did when I followed  an example out of Zaccone's "Getting Started with Tensorflow". 
Hope this works for you !! (;-)
